scripts/create_image.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+mkdir $1
+sudo debootstrap jammy $1  http://archive.ubuntu.com/ubuntu
+
+sudo arch-chroot $1 /bin/bash <<EOF
+
+export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+rm /etc/resolv.conf
+echo 'nameserver 1.1.1.1' > /etc/resolv.conf
+
+apt-get update -y
+apt-get install -y cloud-init openssh-server curl
+cloud-init clean
+
+apt-get install -y linux-modules-extra-5.15.0-25-generic
```

Comment:
ain't that quite outdated kernel?

Reply:
I think we have the tools now to try a newer version and immediately try it out see if it works